### DISCUSSION TITLE
How to add a remote temp sensor directly in the esp

### DISCUSSION THREAD

**Discussion by avanh86** [echavet/MitsubishiCN105ESPHome#111] [General]
I got it running good an a esp32 from seeed studio. 
Im wondering if it is possible to directly add a pvvx thermostat so that de remote temperature is used.
I dont want to use HA to inject the remote temperature.
I already got my pvvx thermostat connected to the esp32 i now want use that temperature.

**Comment by echavet** [comment]
Hello @avanh86,
Could you please provide your yaml?

**Reply by avanh86** [reply]
I got it working. Im sorry for the late response.

**Reply by klim123123** [reply]
Please provide config. I want to do the same.
Thank you
